src/ResourceManager/Network/Commands.Network/help/Set-AzNetworkWatcherConfigFlowLog.md

```diff
@@ -16,72 +16,80 @@ Configures flow logging for a target resource.
 ```
```

Comment:
@zwswim would you mind adding another example to this file that uses the new parameters you are adding and displays the corresponding output?

Reply:
@cormacpayne, 1 example  added for new parameter